src/_lib/modules/word-cloud/word-cloud.component.spec.ts

```diff
@@ -79,6 +79,22 @@ describe('WordCloudComponent', () => {
             expect(fontSize).toBeGreaterThanOrEqual(12);
             expect(fontSize).toBeLessThanOrEqual(60);
         });
+
+        it('should return a font size midway between minFontSize and maxFontSize when all wordWeights are the same', () => {
+            const minWeight = 100;
+            const maxWeight = 100;
+            const wordWeight = 100;
+
+            comp.options = { minFontSize: 25, maxFontSize: 75 };
+            comp.ngOnChanges({
+                words: new SimpleChange(undefined, [], true)
+            });
+            fixture.detectChanges();
```

Comment:
Out of curiosity, why use `onChanges` -> `init` -> `_getTruncatedWordsWithFontSize` just to the get config object?  Could you call `comp.config = comp['_getConfig]()` directly and skip the `ngOnChanges` and `detectChanges` steps?

Reply:
This approach just seemed a bit more natural to me but I think your suggestion might be easier to understand. I'll make that change